{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
Well, okay, so what's the idiomatic way of doing setup/teardown w/ pytest?

Do we leave that to users figure out? Do we assume users that choose to use pytest already know it?